secret-manager/createSecretWithLabels.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
```

Comment:
Nit (for all samples): I don't think it's good practice to have default values since the sample will fail confusingly for users not running it with the correct values. I think its better to leave them empty so that the user gets a message as to why it fails. Plus, you have the sample values in the comments anyways.

Reply:
Fixed